src/lib/Hydra/Helper/AddBuilds.pm

```diff
@@ -57,7 +57,7 @@ sub updateDeclarativeJobset {
                 name => $name,
                 type => $data->{type}
             };
-            $row->{emailresponsible} = $data->{emailresponsible} if defined $data->{emailresponsible};
+            $row->{emailresponsible} = $data->{emailresponsible} or 0;
```

Comment:
Shouldn't this be `// 0`? I think `or` binds very weakly.

Reply:
But that doesn't matter, does it?

Reply:
Doesn't it parse as `($row->{emailresponsible} = $data->{emailresponsible}) or 0;`? I could be wrong.